packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -452,41 +443,54 @@ export const saveClicked: AsyncAction = withOwnedSandbox(
         state.editor.changedModuleShortids.includes(module.shortid)
       );
 
-      await Promise.all(
-        changedModules.map(module => effects.live.saveModule(module))
-      );
-
-      const updatedModules = await effects.api.saveModules(
-        sandbox.id,
-        changedModules
-      );
-
-      updatedModules.forEach(updatedModule => {
-        const module = sandbox.modules.find(
-          moduleItem => moduleItem.shortid === updatedModule.shortid
+      if (state.user?.experiments.comments) {
```

Comment:
Why do we save all modules here? Shouldn't we save only one module?

Reply:
Nevermind, seeing it now! I don't think that we should keep this one, lots of stuff that's not happening here.